src/lib/notify.js

```diff
@@ -69,11 +69,7 @@ const notify = function(metadata, config) {
       return handleNotify(metadata, config.notify);
     } else {
       // multiple notifications endpoints PR #108
-      for (const key in config.notify) {
-        if (config.notify.hasOwnProperty(key)) {
-          return handleNotify(metadata, config.notify[key]);
-        }
-      }
+      return _.map(config.notify, (key) => handleNotify(metadata, key));
```

Comment:
How are we gonna handle notification errors? This might fails silently. Perhaps you can return a `Promise.all()`